content/moderation/verification.md

```diff
@@ -0,0 +1,116 @@
++++
+title = 'Verification'
+weight = 5
++++
+
+Use Google reCAPTCHA v2 to verify your members before permitting them access to your server. Also track IPs of verifying
+users for alt detection.
```

Comment:
```suggestion
Use Google reCAPTCHA v2 to verify your members before permitting them access to your server. Self-hosters may optionally track IPs of verifying
users for alt detection, though this feature is unavailable on the official instance of YAGPDB.
```

Reply:
Imo a bit specific for a summary

Reply:
Yeah, true. My main objection is that claiming "track IPs of verifying users for alt detection" without further context is misleading, since this functionality is not, in fact, available on the primary instance. Perhaps this sentence should be removed entirely?

Reply:
"IPs" -> "IP addresses"